docs/mendix-github-actions.md

```diff
@@ -0,0 +1,131 @@
+- Select "Actions" in the sub-menu that appears below "Secrets"
+- Use the "New repository secret" button
+- Add an first environment variable `SIGRID_CI_TOKEN` and use your [Sigrid authentication token](authentication-tokens.md) as the value.
+- Add an second environment variable `MENDIX_TOKEN` and use this Mendix app [warden.mendix.com](https://warden.mendix.com) to create a PAT with 'mx:modelrepository:repo:read' access only.
+
+<img src="images/github-env.png" width="400" />
+<img src="images/mendix-credentials.png" width="600" />
+
+Your repository secret should now look like this:
+
+<img src="images/github-env-secrets.png" width="400" />
+
+This example explained how to add secrets for a single repository. However, if you have a GitHub organization with many repositories it can be cumbersome to repeat these steps for every repository. You can solve this by adding secrets to your GitHub organization. The process is the same as explained above, though you should access the "secrets" menu for your GitHub organization instead of the "secrets" page for the repository.
+
+<img src="images/github-org-secrets.jpg" width="400" />
+
+The organization-level secret.
+
+**Create a GitHub Actions workflow for Sigrid CI**
+
+We will create a pipeline that consists of two jobs:
+
+- One job that will publish the main branch to [sigrid-says.com](https://sigrid-says.com) after every commit to main.
+- One job to provide feedback on pull requests, which can be used as input for code reviews.
+
+
+We will create two GitHub Action workflows: the first will publish the main/master branch to [sigrid-says.com](https://sigrid-says.com) after every commit. In your GitHub repository, create a file `.github/workflows/sigrid-publish.yml` and give it the following contents:
+
+```
+name: sigrid-publish
+on:
+  push:
+    branches:
+      - "main"
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: softwareimprovementgroup/mendixpreprocessor:latest
+    env:
+      SIGRID_CI_CUSTOMER: 'examplecustomername'
+      SIGRID_CI_SYSTEM: 'examplesystemname'
+      SIGRID_CI_TOKEN: "${{ secrets.SIGRID_CI_TOKEN }}"
+      MENDIX_TOKEN: "${{ secrets.MENDIX_TOKEN }}"
+      CI_PROJECT_DIR: "."
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - run: |
+          echo $SIGRID_CI_TOKEN
+          /usr/local/bin/entrypoint.sh
+
+```
+
+Note the name of the branch, which is `main` in the example but might be different for your repository. In general, most older projects will use `master` as their main branch, while more recent projects will use `main`. 
+
+Next, we create a separate workflow for the pull request integration. This will compare the contents of the pull request against the main/master branch from the previous step. In your GitHub repository, create a file `.github/workflows/sigrid-pullrequest.yml` and give it the following contents:
+
+```
+name: sigrid-pullrequest
+on: [pull_request]
+jobs:
+  container:
```

Comment:
you can rename 'container' into SigridCI-for-QSM

Reply:
Renamed